gluten-core/src/main/scala/org/apache/spark/sql/execution/datasources/GlutenWriterColumnarRules.scala

```diff
@@ -144,7 +145,16 @@ object GlutenWriterColumnarRules {
   }
 
   case class NativeWritePostRule(session: SparkSession) extends Rule[SparkPlan] {
+
+    private val NOOP_WRITE = "org.apache.spark.sql.execution.datasources.noop.NoopWrite$"
```

Comment:
is this class name valid for spark 32/33/34 ?

Reply:
@zhouyuan The class name has been verified to exist in Spark [3.2](https://github.com/apache/spark/blob/v3.2.2/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/noop/NoopDataSource.scala#L61) and [3.3](https://github.com/apache/spark/blob/v3.3.2/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/noop/NoopDataSource.scala#L59). However, starting from Spark 3.4, this code will no longer be executed.